Corona-Warn-App/src/main/res/values-en/strings.xml

```diff
@@ -852,10 +852,10 @@
     <string name="debugging_debuglog_action_start_recording">"Start"</string>
     <!-- XBUT: Button text to stop the log recording -->
     <string name="debugging_debuglog_action_stop_recording">"Stop and Delete"</string>
-    <!-- XBUT: Button text to share the log recording -->
+    <!-- XBUT: Button text to send the log recording to the server -->
     <string name="debugging_debuglog_action_share_log">"Send Error Report"</string>
-    <!-- XBUT: Button text to locally store the log recording -->
-    <string name="debugging_debuglog_action_local_log_store">"Save Locally and Continue"</string>
+    <!-- XBUT: Button text to export the log -->
+    <string name="debugging_debuglog_action_local_log_export">"Save Locally and Continue"</string>
```

Comment:
should not we change that? it does not save anymore?

Reply:
Valid point, I'll ping @janetback .

Reply:
Discussed with @janetback changed export to share, will be part of the next translation delivery.

Reply:
Yes, I only changed the german value according to the jira story description and expected the other languages in the next translation delivery.